math/fibonacci_matrix_exponentiation.cpp

```diff
@@ -0,0 +1,143 @@
+ /**
+ * @file fibonacci_matrix_exponentiation.cpp
+ * @brief Computes N^th Fibonacci number given as
+ * input argument
+ *
+ * Takes O(logn) time to compute nth Fibonacci number
+ * 
+ *
+ * \author [villayatali123](https://github.com/villayatali123)
+ * \author [unknown author]()
+ * @see fibonacci.cpp, fibonacci_fast.cpp, string_fibonacci.cpp, fibonacci_large.cpp
+ * @details
+ *Begin 
+ * Take two 2 dimensional vector
+ * Create a function and Perform matrix multiplication
+ * Ceate another function to find out power of matrix
+ * Create a function then to find out the Fibonacci number
+ * Multiply(arr1[2][2], arr2[2][2], int size)
+ * Take 4 variables a, b, c, d
+ * ''' 	
```

Comment:
as suggested before, the character to mark a verbatim section is: [`&grave;`](https://www.toptal.com/designers/htmlarrows/punctuation/grave-accent/) and not [`'`](https://www.toptal.com/designers/htmlarrows/punctuation/apostrophe/)